telebot/__init__.py

```diff
@@ -2972,15 +2977,31 @@ def restrict_chat_member(
             can_send_voice_notes permissions; the can_send_polls permission will imply the can_send_messages permission.
         :type use_independent_chat_permissions: :obj:`bool`
 
+        :param permissions: Pass ChatPermissions object to set all permissions at once. Use this param instead of
+            passing all boolean parameters.
+        :type permissions: :class:`telebot.types.ChatPermissions`
+
         :return: True on success
         :rtype: :obj:`bool`
         """
+
+        if permissions is None:
+            permissions = types.ChatPermissions(
+                can_send_messages=can_send_messages,
+                can_send_media_messages=can_send_media_messages,
+                can_send_polls=can_send_polls,
+                can_send_other_messages=can_send_other_messages,
+                can_add_web_page_previews=can_add_web_page_previews,
+                can_change_info=can_change_info,
+                can_invite_users=can_invite_users,
+                can_pin_messages=can_pin_messages
+            )
+            logger.warning(
+                'Please pass `telebot.types.ChatPermissions` object to `permissions` parameter instead of '
```

Comment:
I propose to use "Individual parameters are deprecated and will be removed, use 'permissions'" instead.